ano/environment.py

```diff
@@ -372,8 +372,7 @@ def process_args(self, args):
         if arduino_dist:
             distHash = hashlib.md5(arduino_dist).hexdigest()[:8]
             build_dirname = '%s-%s' % (build_dirname, distHash)
-
-        self['build_dir'] = os.path.join(self.output_dir, build_dirname)
+            self['build_dir'] = os.path.join(self.output_dir, build_dirname)
```

Comment:
Not quite it; this causes an AttributeError for some cause. That said; this give me a lot more data. I might be able to isolate the problem now.